NuRadioReco/modules/io/eventWriter.py

```diff
@@ -98,7 +98,9 @@ def run(self, evt, det = None, mode='full'):
             self.__current_file_size = 0
             self.__fout.close()
             self.__number_of_files += 1
-            self.__filename = "{}_part{:02d}.nur".format(self.__filename, self.__number_of_files)
+            print('!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!')
```

Comment:
can you remove the print statement and use the logger instead?

Reply:
Done